src/main/java/frc/robot/subsystems/GroundSpinny.java

```diff
@@ -80,6 +81,10 @@ public Command holdFunnelIntakePower() {
     return holdPower(FUNNEL_INTAKE_SPEED).withName("hold funnel intake power");
   }
 
+  public Command holdCoralPower() {
+    return holdPower(GROUND_INTAKE_HOLD_SPEED).withName("hold coral holding power");
```

Comment:
What is the impact on battery for this?  Is there risk of pulling too much from the battery limiting other actions?  Does this reduce battery life?